test/concentrate/consumer/stop_time_updates_test.exs

```diff
@@ -39,6 +55,9 @@ defmodule Concentrate.Supervisor.StopTimeUpdatesTest do
 
   describe "handle_events/3" do
     setup do
+      reassign_env(:realtime, :trip_fn, fn _trip_id -> @trip end)
+      reassign_env(:realtime, :block_fn, fn _block_id, _service_id -> nil end)
```

Comment:
If the mocked block function doesn't return a block, what's this testing? Should it return `[@trip]`?